packages/venia-concept/src/components/MyAccountMenuPage/MenuItem/__tests__/menuItem.spec.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import MenuItem from '../menuItem';
+
+configure({ adapter: new Adapter() });
```

Comment:
We shouldn't need `Adapter` or `configure` anymore. Since this PR was created, they've been applied globally.